src/component/attendance/Header.js

```diff
@@ -0,0 +1,43 @@
+import { Box } from '@chakra-ui/layout'
+import {React, useState} from 'react'
+import bell from '../../assets/notification-bell.svg'
+import bell2 from '../../assets/smart-bell.svg'
+import {Center, Divider, Image} from '@chakra-ui/react'
+import {ListItem,UnorderedList} from '@chakra-ui/react'
+
+
+
+const Header = () => {
+    const [state, setstate] = useState("hidden");
+    const [bellState, setBellState] = useState(bell);
+    let messages= ['There will be a general public holiday tomorrow Friday 13/10/2021.',
```

Comment:
The messages are being hard coded here, They should be passed in as props, to make this component more reusable